src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -298,7 +298,7 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     files = {('/' + cpi['publicKeyIdentifierFile']):
              cpi['publicKeyIdentifierFile']
              for cpi in config['cpiDatabaseConfig']['cpis']}
-    files['/' + config['cpiDatabaseConfig']['indexPath']] = config[
+    files['/' + config['cpiDatabaseConfig']['indexUrl']] = config[
```

Comment:
Sorry I didn't catch it earlier, but the leading "/" should be removed from this line since it is already present in the index URL.

Reply:
yes, you are right. thanks.